pipeline.yml

```diff
@@ -16,18 +16,25 @@ schedule :
 build:
   -
     scripts:
-      - "dotnet build --configuration Release IronPython3Extension.sln"
+      - dotnet build --configuration Release IronPython3Extension.sln
 
 ci_test:
   tests:    
     -
       scripts:
-        - "dotnet test --no-build --configuration Release IronPython3Extension.sln --filter \"TestCategory!=Failure\" --logger:junit"   
+        - dotnet coverage collect "dotnet test --no-build --configuration Release IronPython3Extension.sln --filter TestCategory!=Failure --logger:junit  --logger:trx" -f xml -o "coverage.xml"
```

Comment:
You need add back \" for the filters again. Sorry about that.

Reply:
That's ok, done!